articles/solving-linear-equations-using-matlab/index.md

```diff
@@ -0,0 +1,211 @@
+```Matlab
+A = [4 5]; b = 6;
+x = A\b
+```
+The output will be;
+```
+x =
+
+         0
+    1.2000
+```
+Inadequate information not only occurs in cases where m=n, but it can also occur for problems in which m>n. It could be due to the redundancy of other equations. e.g
+$$
+\begin{cases} 4x + 5y=6\\8x + 10y=12\\ -2x -2.5y =-3\end{cases}
+$$
+In the above equation, the information is insufficient since equations (i) and (ii) can be generated from equation (i). If you take equation(i) and multiply by 2, we get equation (ii), and also, if you take equation(i) and divide by -2, we get equation(iii).
+
+### Overdetermined problems
+This kind of problem mainly occurs when m>n. Here, the information provided by the equations is too much. The equation Ax=b cannot be satisfied simultaneously by any value of the vector x. When you solve the equation using Matlab, it will give an output, but it does not satisfy the matrix rule a*x =b. To understand this, we start by plotting the equations below;
+$$
+\begin{cases} 4x + 5y=6\\3x - 2y=14\end{cases}$$
+For the first equation, y1 = -(4x - 6)/5. To plot `y2` and `y1` against `x`.
+```matlab
+x = [-10;10];
+y1 = (-4*x + 6)/5;
+y2 = (3*x -14)/2;
+plot(x, y1, x, y2);
+grid on
+xlabel('x');
+ylabel('y');
+legend('y1', 'y2')
+```
+![output two](/engineering-education/solving-linear-equation-using-matlab/linear_two.png)
+We will use the `intersect function` to find the point of intersection to find the point at which the lines intersects. The intersect function uses the gradients and the constants as the inputs as shown below;
+```matlab
+function [x0 y0] = intersectPoints(m1,m2,b1,b2) %m is the gradient while b is the constant. 
+x0 = (b2-b1)/(m1-m2); %find the x point
+y0 = m1*x0+b1;
+end
+```
+To call this function in the command window;
+```matlab
+intersectPoints(-4/5, 3/2, 6/5, -14/2) 
+```
+the output will be
+```
+ans =
+
+    3.5652
+```
+This function gives a single output corresponding to the `x` value for the point of intersection. To find the `y`, you can replace the `x` in the equation and get your `y`.
+The diagram shows that the solution is similar to that we found before(3.5652,-1.6522). It means that the solution is correct. Now assume that now we have the equation below;
+$$
+\begin{cases} 4x + 5y=6\\8x + 10y=12\\ -2x -2.5y =-3\\ 7x - y =25\end{cases}
+$$
+Let's plot the line of the three equations. To do that, we add the below code;
+```Matlab
+y3 = 7*x - 25;
+hold on
+plot(x, y1, x,y2, x, y3)
+```
+![output](/engineering-education/solving-linear-equation-using-matlab/linear_three.png)
+When you execute the above commands, we get a 3rd line which is for the third equation. Looking at the plots, you may think these three lines intersect at a common point, but this is not the case. To see this, zoom out the plots. To zoom out, click on the magnifying lens icon having `+` inside it. After clicking it, move to the point at which the lines seems to intersect a few times, and the result will be;
```

Comment:
According to your screenshots(plot output), it has lines with different colors. Can you mention the color of the 3rd line you're describing above. So anyone can easily trace through your explanations for better understanding